src/app/announcement/components/announcement/announcement.component.scss

```diff
@@ -0,0 +1,34 @@
+$banner-bg: #fff3cd;
+$banner-text: #856404;
+
+.announcement {
+  width: 100%;
+  padding: 0.5rem 0.25rem;
+  font-size: 1.25em;
+  background-color: $banner-bg;
+  color: $banner-text;
+}
+
+.announcement-content {
+  margin: 0 auto;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 1em;
+}
+
+.announcement-text {
+  flex-grow: 1;
+  max-width: 80%;
+  margin-right: 1ex;
```

Comment:
In my head, `1ex` approximately equals the length of a space (probably more of an em space maybe) when used for this kind of spacing. I'm just using it here so there's always a bit of room between the text and the dismiss button, and I"m only putting it on the right because the button is always going to be to the right of the text.

Reply:
Thanks, that makes sense to me! And the `max-width: 80%` means that it's unlikely to be a relevant constraint except under rather absurd circumstances, anyway.